src/app/modules/board/client/tasks-client.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { catchError } from 'rxjs/operators';
+import { throwError } from 'rxjs';
+import { SnackBarService } from '../../../shared/snack-bar.service';
+import { environment } from '../../../../environments/environment';
+import type { Task } from '../../../core/models/task';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TasksClientService {
+  private apiURL = environment.apiURL;
```

Comment:
```suggestion
  private apiURL = `${environment.apiURL}/tasks;
```
wyjaśnienie niżej \/ w environment.ts